sigridci/sigridci/objective.py

```diff
@@ -69,10 +70,26 @@ def meetsFindingObjective(findingSeverities, objective):
     @staticmethod
     def isFindingIncluded(severity, objective):
         if objective == "CRITICAL":
-            return severity in ("CRITICAL")
+            return False
         elif objective == "HIGH":
-            return severity in ("CRITICAL", "HIGH")
+            return severity in ("CRITICAL")
         elif objective == "MEDIUM":
+            return severity in ("CRITICAL", "HIGH")
+        elif objective == "LOW":
             return severity in ("CRITICAL", "HIGH", "MEDIUM")
         else:
-            return True
+            return severity in ("CRITICAL", "HIGH", "MEDIUM", "LOW")
+
+    @staticmethod
+    def getSeverityObjectiveLabel(objective):
+        # We phrase objectives for findings as the "worst" severity
+        # that is still allowed. So an objective of HIGH means high-severity
+        # findings are allowed, but critical-severity findings are not allowed.
+        # In the feedback, we want to phrase this in terms of goal, i.e. the
+        # "worst" severity that is *not* allowed.
```

Comment:
```suggestion
        # "least-worst" severity that is *not* allowed.
```
Reason: suppose your objective is `MEDIUM`, which means that the worst severity you still accept is `MEDIUM`. That implies that you don't accept either `HIGH` or `CRITICAL`. The worst severity we don't accept is `CRITICAL`, the "least-worst" is `HIGH`.